deploy/crds/mellanox.com_v1alpha1_nicclusterpolicy_cr.yaml

```diff
@@ -18,6 +18,18 @@ metadata:
   namespace: mlnx-network-operator
 spec:
   ofedDriver:
-    image: "ofed-driver"
-    repository: "adriancdockerhub"
-    version: "5.0-2.1.8.1"
+    image: ofed-driver
+    repository: adriancdockerhub
```

Comment:
note: to fix this before merge.

Reply:
i will change to some generic value like

REPLACE_REPOSITORY
REPLACE_IMAGE
REPLACE_VERSION

then when all is working i can set the example to point to whatever we upload to mellanox dockerhub 
WDYT ?

Reply:
sure let do this